packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -173,7 +173,8 @@ - (void)launchImagePicker:(EXImagePickerTarget)target
     if (@available(iOS 11.0, *)) {
       self.picker.videoExportPreset = [self importVideoExportPreset:self.options[@"videoExportPreset"]];
     }
-    
+    self.picker.videoQuality = UIImagePickerControllerQualityTypeHigh;
```

Comment:
There might be some cases where people don't want to use the highest quality — it would be better to provide a new option to control this behavior.
cc @lukmccall

Reply:
In my understanding this should still be effected by the `VideoExportPreset` option, so the saved video does not have to be high quality. Although `VideoExportPreset` is being deprecated in iOS 14, so maybe it should be replaced with [UIImagePickerControllerQualityType](https://developer.apple.com/documentation/uikit/uiimagepickercontrollerqualitytype), which is used here.

Reply:
Maybe we could deprecate `videoExportPreset` and switch to `UIImagePickerControllerQualityType`? Or maybe it will be better if we change `picker.videoQuality` according to the `options[@"videoExportPreset"]`. I'm not sure which solution is better in this case.

Reply:
I was considering the latter, but I often end up looking at Apple documentation, so using the `VideoExportPreset` value for another valid iOS option seems confusing. I prefer going with the first option if that's fine.

Reply:
Ok, it sounds great 🎉 Thanks for taking care of it ;)